client/src/app/home/feature/home/home.page.html

```diff
@@ -167,12 +167,22 @@ <h2>See who reviewed the cards</h2>
   <div class="col-12 flex-order-1 lg:flex-order-2">
     <ng-container *ngIf="{loadingUsers: loadingUsers | async} as context">
       <p-table
-        *ngIf="usersWithRating[0]?.totalDeviation"
-        [value]="usersWithRating"
+        *ngIf="filteredUsersWithRating[0]?.totalDeviation"
```

Comment:
the conditional should still use `usersWithRating` to verify the format of the data, and make sure it has totalDeviation (meaning it is post-expansion reviews). The filtered users could be empty, therefore the condition would be false,